README.md

```diff
@@ -70,6 +70,7 @@ sway-applications/
 
 - [Name-Registry](./name-registry/) allows users to perform transactions with human readable names instead of addresses.
 - [Oracle](./oracle) is a smart contract that provides off-chain data to on-chain applications.
+- [Counter-Script](./counter-script/) is a tutorial for using scripts.
```

Comment:
```suggestion
- [Counter-Script](./counter-script/) is a script that calls a contract to increment a counter.
- [Name-Registry](./name-registry/) allows users to perform transactions with human readable names instead of addresses.
- [Oracle](./oracle) is a smart contract that provides off-chain data to on-chain applications.
```